docs/pages/development/getting-started.md

```diff
@@ -7,16 +7,16 @@ import SnackInline from '~/components/plugins/SnackInline';
 import { Tab, Tabs } from '~/components/plugins/Tabs';
 import { Terminal } from '~/ui/components/Snippet';
 
-Development builds of your app are Debug builds of your project that include the [`expo-dev-client`](https://www.npmjs.com/package/expo-dev-client) library, which allows you to develop and debug projects from expo-cli or a compatible server.
+Development builds of your app are Debug builds of your project that includes the [`expo-dev-client`](https://www.npmjs.com/package/expo-dev-client) library. It allows you to develop and debug projects from Expo CLI or a compatible server.
```

Comment:
'includes' reads weird here. Is it the builds that include it, or the project?

Reply:
🤔 I can see the wording being confusing. I think it will be more clear to say that Development builds use `expo-dev-client` since the library doesn't come included in the project and one has to manually install it.